packages/ra-supabase-core/src/authProvider.ts

```diff
@@ -84,7 +84,7 @@ export const supabaseAuthProvider = (
             if (type === 'recovery' || type === 'invite') {
                 if (access_token && refresh_token) {
                     return {
-                        redirectTo: `set-password?access_token=${access_token}&refresh_token=${refresh_token}&type=${type}`,
+                        redirectTo: `${redirectTo}/set-password?access_token=${access_token}&refresh_token=${refresh_token}&type=${type}`,
```

Comment:
I'm afraid this might include `undefined` in the string. Maybe add a default empty string as the `redirectTo` default?

Reply:
Hi, thanks for your fast review. I have updated the code: using a nullish operator (`??`) in the string interpolation, the `undefined` and `null` cases can be properly controlled.